RT8_OP_CODES.s

```diff
@@ -136,7 +136,7 @@ TPMODE		CAF	ONE		# MODE IS TP.
 # Page 1510
 		TC	Q
 
-		INDEX	A		# OVERFLOW UNCORRECT AND IN MSU.
+		INDEX	A		# OVERFLOW INCORRECT AND IN MSU.
```

Comment:
This is `UNCORRECT` in the scan of [page 1510](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1510.jpg).